sync/wizard/sync_make_module.py

```diff
@@ -140,14 +143,15 @@ def act_makefile(self):
                         (
                             "trigger_name",
                             "active",
+                            "name",
```

Comment:
I skip `name` because it mostly duplicates information in trigger_name. I'd probably remove the field completly. So, I'm not sure about proposed change